packages/admin/src/App/Home/Products.js

```diff
@@ -23,8 +26,23 @@ class Products extends React.Component {
     this.setState({ products, loading: false });
   };
 
+  handleDeleteProd = async () => {
+    this.setState({ isDeletingProd: true });
+    await deleteProduct(this.state.selectedProd);
+
+    this.setState(prev => ({
+      products: { ...prev.products, data: prev.products.data.filter(prod => prod.id !== this.state.selectedProd) },
```

Comment:
I suggest not to do the delete locally because if we have pagination, not all the products will be loaded at once and this will break.
Therefore, I suggest just to call loadProducts to reload the products and refresh the state